problem 1 prompt and one possible solution (#42)
UWB-ACM/CTCI: Spring-2019/system_design_misc_coding/SOLUTIONS_2019Spring.md

## Problems

### 1. Flight Reservations

Source: Tom Blanchard

#### Scenario

> A plane with 300 seats; 50 first class seats, 50 comfort seats and 200 economy seats

Author: ```suggestion
A plane has 300 seats; 50 first class seats, 50 comfort seats and 200 economy seats.
```

> must have a way for customers to reserve thier seats. How would you design a system for 

Author: ```suggestion
The airline must have a way for customers to reserve their seats. How would you design a system for 
```

## Solutions

### 1. Flight Reservation UML Class Diagram
This diagram can be located in `Spring-2019/system_design_misc_coding/Flight_Reservation/Flight_Reservation_System.png`

> 

Author: ```suggestion
![Reservation Diagram](./Flight_Reservation/Flight_Reservation_System.png)
```
